infra/gcp/terraform/k8s-infra-gcp-gcve/docs/terraform.md

```diff
@@ -19,7 +19,7 @@ You can use terraform from your local workstation or via a docker container prov
 docker run -it --rm -v $(pwd):/workspace --entrypoint=/bin/bash gcr.io/k8s-staging-infra-tools/k8s-infra:v20241217-f8b07a049
 ```
 
-From your local workstatin / from inside the terraform container:
+From your local workstation / from inside the terraform container:
 
 Login to GCP to get an authentication token to use with terraform.
 
```

Comment:
Did l.27 just work for you?

I needed some combination of the following commands to get it to work
```bash
gcloud auth login
gcloud config set account xxx
gcloud config set project xxx
```
(not sure what the minimum set of commands is)